### PR TITLE
Update 5_References.md

### PR DESCRIPTION
**Pull request by boshnivolo**

### CODE REVIEW

**Comment by Ricecoder** [comment]
Your journal, references, and tutorial are both funny and slightly bitter but like dark chocolate it works! Add some more text to your amazing gifs and pictures and you will be good to go :+1: :+1: :+1:

**Comment by jmcreasman** [comment]
Tutorial:
Best tutorial ever! :+1: 
Reflective:
I'm with ya on the chess so glad to have that one behind me :+1: 
Resource:
I was looking for this video so I'm glad you linked it - I have 3+ hours to give :+1:

**Comment by jeffworley** [comment]
Looks solid!  Dig you injection of personality into these. :+1:

File: 5_Revenge_of_the_Chessboard.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+<img src="http://i.imgur.com/bJcII6E.gif" />
+<img src="http://i.imgur.com/ySDkC87.gif" />
 
-MUCH 
+Ah yes, like M. Night Shyamalan movies and the flu, the Chessboard assignment was inflicted upon us again this week. 
 
-CHESSBOARDDDD
+<img src="http://static.socialitelife.com/uploads/2013/10/11/crying-gifs-10112013-09.gif" />
+
+This week though I decided that the only thing that could defeat this great evil was something just as vile and horrid...<strong>MATH</strong>.
```
Comment (kellymurray) [inline 5_Revenge_of_the_Chessboard.md:12]: :100:

**Pull request merged**